cfgov/data_research/blocks.py

```diff
@@ -17,9 +17,21 @@ class ConferenceRegistrationForm(AbstractFormBlock):
     capacity = atoms.IntegerBlock(help_text=(
         'Enter an integer that will be the conference attendance limit.'
     ))
-    at_capacity_message = organisms.FullWidthText(help_text=(
-        'Enter a message that will be shown when the event is at capacity'
-    ))
+    success_message = blocks.RichTextBlock(
+        help_text=(
+            'Enter a message that will be shown on successful registration.'
+        )
+    )
+    at_capacity_message = organisms.FullWidthText(
+        help_text=(
+            'Enter a message that will be shown when the event is at capacity.'
+        )
+    )
+    failure_message = blocks.CharBlock(
+        help_text=(
+            'Enter a message that will be shown on registration failure.'
```

Comment:
@cfarm I'm happy to change it if you have an alternate proposal. Yes, right now the error only appears if our [call](https://github.com/cfpb/cfgov-refresh/pull/2548/files#diff-2ab90947aaf17de9ab23cdbff150638fR35) to subscribe to GovDelivery fails. This might happen if GD isn't configured properly (through, e.g. environment variables) or if something goes wrong with the GD API.

From the user's point of view, though, all they know is that something went wrong on our end. There's really nothing they can do besides try to resubmit at some later date. Unfortunately we wouldn't even really know that something went wrong unless we monitored the logs for these errors or if users let us know of issues they were seeing.